mediawiki-scraper/launcher.py

```diff
@@ -25,8 +25,8 @@
 import time
 from pathlib import Path
 
-from wikiteam3.dumpgenerator.config import Config
-from wikiteam3.utils import domain2prefix
+from mediawiki-scraper.dumpgenerator.config import Config
```

Comment:
Can you test to make sure the hyphenated version of the name is correct? IIRC package names have to use underscores rather than hyphens. (Basically just run a sample command from the command line if you don't have an interactive development environment set up.)

Reply:
I'll check but it will take a little while, I'm getting out of my depth here. Won't it mess things up if file names are hyphens and package names are underscores?